SpatialGDK/Source/SpatialGDK/Private/SpatialView/SubView.cpp

```diff
@@ -55,18 +55,9 @@ const FSubViewDelta& FSubView::GetViewDelta() const
 	return SubViewDelta;
 }
 
-TSet<Worker_EntityId_Key> FSubView::GetCompleteEntities() const
+TArray<Worker_EntityId> FSubView::GetCompleteEntities() const
```

Comment:
Do we maybe want to enhance this up a bit by returning a `const TArray<Worker_EntityId>&`, less copies this way?